src/NCPopupMenu.cc

```diff
@@ -141,3 +171,79 @@ bool NCPopupMenu::postAgain()
     return again;
 }
 
+
+NCursesEvent NCPopupMenu::select_next_item()
+{
+    NCursesEvent event;
+
+    YTableItem * tableItem = dynamic_cast<YTableItem *> ( getCurrentItemPointer() );
+
+    if ( !tableItem)
+	return event;
+
+    auto current = find_item(tableItem);
+
+    if ( current == _items.end() )
+	return event;
+
+    auto begin = std::next(current, 1);
```

Comment:
Those `auto` orgies make the code hard to read. What type are we dealing here?
The reader of the code has to do the compiler's job to figure that out. Please don't do that.